### ISSUE TITLE
Wrong number of releases in project overview

### ISSUE THREAD

**Issue by AX502** [eclipse/sw360#775]
As you can see the UI shows different information on the home screen and the project overview.
I have two releases linked to my project and only one is cleared. Somehow the other failed its upload to FOSSology and is stuck in clearing state 'Sent to clearing tool'.
Maybe the release is not recognized as long as it remains in this clearing state.

![issue_project_view](https://user-images.githubusercontent.com/40486005/74216437-c31c9800-4ca4-11ea-8c27-e25d0577c1d4.PNG)
![issue_home_view](https://user-images.githubusercontent.com/40486005/74216440-c4e65b80-4ca4-11ea-96be-91cf2c61dcfb.PNG)

**Comment by mcjaeger** [comment]
? I am not sure if I entirely understand the issue. So in other words:
* number of total releases is shown in project listing
* number of total releases is shown in "my projects" portly shown on home screen
-> should be the same.

I am not suspecting the FOSSology part here, but maybe caching issues when the view is computed. I think the there is a DB view for that? Maybe it is a caching issue

**Comment by AX502** [comment]
> 
> 
> ? I am not sure if I entirely understand the issue. So in other words:
> 
>     * number of total releases is shown in project listing
> 
>     * number of total releases is shown in "my projects" portly shown on home screen
>       -> should be the same.
>
That's right, the first picture shows the project listing and the second one shows the "my projects" portly on home screen.
